CMakeLists.txt

```diff
@@ -1221,6 +1225,17 @@ else()
         set_target_properties(${IMPL_TARGET} PROPERTIES PREFIX "")
     endif()
     target_link_libraries(${IMPL_TARGET} PRIVATE common ${LINKER_FLAGS} ${EXTRA_LIBS} ${MATH_LIB})
+
+    if(APPLE AND ALSOFT_OSX_FRAMEWORK)
+        set_target_properties(${IMPL_TARGET} PROPERTIES
+            FRAMEWORK TRUE
+            FRAMEWORK_VERSION C
+            MACOSX_FRAMEWORK_IDENTIFIER org.opena-soft.openal
```

Comment:
L character is missing in the domain name.

Reply:
fixed